pkg/insights/insightsreport/insightsreport.go

```diff
@@ -38,10 +38,13 @@ type InsightsReporter interface {
 	ArchiveUploaded() <-chan struct{}
 }
 
-var (
+const (
+	insightsLastGatherTimeName = "insights_last_gather_time"
+)
```

Comment:
This is renaming the existing metric. Please preserve the original name `insightsclient_last_gather_time` since there's already functionality depending on this metric.

Reply:
Did the name of the metric change, by any chance? Because I'm pretty sure I just copy-pasted the string, so I don't understand why it would be any different from what it originally was, but sure, I'll change that.

Reply:
You changed it. See:
- https://github.com/openshift/insights-operator/pull/607/files#diff-d543de2f94800472455800fb3d022ea29cc4d84486bc1ca6a9e2b7d36ae37124L55 
- https://github.com/openshift/insights-operator/pull/607/files#diff-d543de2f94800472455800fb3d022ea29cc4d84486bc1ca6a9e2b7d36ae37124R42

Reply:
@tremes It wouldn't be visible in the commit history if the name was changed in parallel to my changes because I would have rebased my commits on top of the original name change. It's pretty weird, but whatever.